app/static/css/custom/footer.css

```diff
@@ -53,6 +52,10 @@ footer .copyright{
 
 #footer-menu select {background: #282C35; border-color:#AAAAAA; color: #AAAAAA  }
 
+#footer-menu-options{
```

Comment:
@PARAG00991 Don't create a new ID for just this ... Refer the div from the already existing  `#footer` ID